event-sourcing/event-source/src/main/java/uk/gov/justice/services/eventsourcing/source/core/exception/AggregateCreationException.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.justice.services.eventsourcing.source.core.exception;
+
+public class AggregateCreationException extends RuntimeException {
+
+    public static final String ACCESS_ERROR = "Unable to create aggregate due to access error";
```

Comment:
I think it might be better that the message is provided in the constructor and not as constants.  The constructor can then be generalized to one constructor.